benchmark-service/benchmark/benchmark.py

```diff
@@ -41,12 +51,21 @@ async def index(request: web.Request) -> Dict[str, Any]:
 
 def init_app() -> web.Application:
     app = web.Application()
+    setup_aiohttp_jinja2(app, 'benchmark')
+    setup_aiohttp_session(app)
+    setup_common_static_routes(router)
+    admin = web.Application()
     app.add_routes(router)
+    # admin.add_subapp('/dabuhijl/benchmark/', app)
     aiohttp_jinja2.setup(
```

Comment:
You don't need to explicitly setup aiohttp_jinja2 here anymore because you're calling a wrapper function above `setup_aiohttp_jinja2` that does that for you.